### PR TITLE
add storybook for options from an object

### PR DESCRIPTION
**Pull request by thaydds**
#### What does this PR do?
add a simple storybook about how to use an object in a select option

#### Where should the reviewer start?
Select/stories/OptionsFromObject

#### Any background context you want to provide?
no

#### What are the relevant issues?
#3679 

#### Do the grommet docs need to be updated?
i dont think so, another examples have a select with options provide for an object but this simple example that i made was pretty objective.

#### Should this PR be mentioned in the release notes?
no

#### Is this change backwards compatible or is it a breaking change?
compatible

### CODE REVIEW

File: src/js/components/Select/stories/OptionsFromObject.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        labelKey="lab"
+        value={value}
+        options={objectOptions}
+        onChange={({ option }) => setValue(option)}
+      />
+    </Box>
+  );
+};
+
+storiesOf('Select', module).add('OptionsFromObject', () => (
```
Comment (oorestisime) [inline src/js/components/Select/stories/OptionsFromObject.js]: Hey there,

thanks for this story. can you change the label here with spaces? this is publicly available on storybook so it should be easily readable. Maybe Object options would also fit better. other wise good to go

Comment (ShimiSun) [reply src/js/components/Select/stories/OptionsFromObject.js]:
```suggestion
storiesOf('Select', module).add('Options from object', () => (
```

**Comment by mrcleanandfresh** [comment]
I could have really used an example like this when I was looking through this documentation! It would have saved me from having to bother the maintainers on Slack. :)

File: src/js/components/Select/stories/OptionsFromObject.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Box, Select } from 'grommet';
+
+const objectOptions = [
+  {
+    lab: 'Male',
+    val: 1,
```
[the remark this answers is not in the document]
Comment (mrcleanandfresh) [reply src/js/components/Select/stories/OptionsFromObject.js]: If I may contribute an opinion here. Personally, I like to not abbreviate as much as possible in examples. So I would have put `label` and `value` just to be explicitly clear, especially for developers that may not know the abbreviation. Like I said, that's just my personal preference.

[the remark this answers is not in the document]
Comment (ShimiSun) [reply src/js/components/Select/stories/OptionsFromObject.js]: @mrcleanandfresh , great feedback, thanks for sharing!

[the remark this answers is not in the document]
Comment (oorestisime) [reply src/js/components/Select/stories/OptionsFromObject.js]: yeap agreed!

[the remark this answers is not in the document]
Comment (ShimiSun) [reply src/js/components/Select/stories/OptionsFromObject.js]:
```suggestion
    value: 1,
```

File: src/js/components/Select/stories/OptionsFromObject.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  {
+    lab: 'Female',
+    value: 2,
+  },
+  {
+    lab: 'Non Binary',
```
Comment (ShimiSun) [inline src/js/components/Select/stories/OptionsFromObject.js]:
```suggestion
    label: 'Non Binary',
```

File: src/js/components/Select/stories/OptionsFromObject.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+import { storiesOf } from '@storybook/react';
+import { Box, Select } from 'grommet';
+
+const objectOptions = [
+  {
+    lab: 'Male',
+    val: 1,
+  },
+  {
+    lab: 'Female',
```
Comment (ShimiSun) [inline src/js/components/Select/stories/OptionsFromObject.js]:
```suggestion
    label: 'Female',
```

File: src/js/components/Select/stories/OptionsFromObject.js (hunk 2 above)
Comment (ShimiSun) [inline src/js/components/Select/stories/OptionsFromObject.js]:
```suggestion
    label: 'Male',
    value: 1,
```

File: src/js/components/Select/stories/OptionsFromObject.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    value: 2,
+  },
+  {
+    lab: 'Non Binary',
+    value: 3,
+  },
+  {
+    lab: 'Other',
```
Comment (ShimiSun) [inline src/js/components/Select/stories/OptionsFromObject.js]:
```suggestion
    label: 'Other',
```

File: src/js/components/Select/stories/OptionsFromObject.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+const OptionsFromObject = () => {
+  const [value, setValue] = useState('');
+
+  return (
+    <Box fill align="center" justify="start" pad="large">
+      <Select
+        id="select"
+        name="select"
+        placeholder="Select"
+        labelKey="lab"
```
Comment (ShimiSun) [inline src/js/components/Select/stories/OptionsFromObject.js]:
```suggestion
        labelKey="label"
```

File: src/js/components/Select/stories/OptionsFromObject.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Box, Select } from 'grommet';
+
+const objectOptions = [
+  {
+    lab: 'Male',
```
Comment (ShimiSun) [inline src/js/components/Select/stories/OptionsFromObject.js]:
```suggestion
    label: 'Male',
```

**Pull request merged**